### PR TITLE
coding style improvments + *.pro file for mac os

### PR DESCRIPTION
**Pull request by woskov**
resloving code-style issuses and pro file for mac

### CODE REVIEW

File: src-gui/meshpeer.h [hunk 1]
```diff
@@ -18,11 +18,11 @@ class MeshPeer : public QObject
 
     enum COMMANDSTATUS {
         none = 0,
-        sended,			//wyslano do noda galaxy
-        recived,		//otrzymano potwierdzenie z sieci galaxy
-        acknowled,		//zsynchronizowano w sieci galaxy
-        timeout,		// timeout komendy
-        fail			//nieudane wykonnie komendy na no
+        sended,
```
Comment (mikurys) [inline src-gui/meshpeer.h]: typo: sent

File: src-gui/meshpeer.h [hunk 2]
```diff
@@ -18,11 +18,11 @@ class MeshPeer : public QObject
...
+        recived,
```
Comment (mikurys) [inline src-gui/meshpeer.h]: typo: received

File: src-gui/meshpeer.h [hunk 3]
```diff
@@ -18,11 +18,11 @@ class MeshPeer : public QObject
...
+        acknowled,
```
Comment (mikurys) [inline src-gui/meshpeer.h]: typo: acknowledged

File: src-gui/commandsender.cpp [hunk 4]
```diff
@@ -99,19 +99,19 @@ std::shared_ptr<order> CommandSender::prepareCommand(CommandSender::orderType ty
         case orderType::ADDPEER:
         if(m_client_name.size() == 0)
             throw std::runtime_error ("no client name provided - can't send command");
-              ord = std::make_shared<addPeerOrder> (new addPeerOrder(m_counter.getRpcId(),peer));
+			  ord = std::make_shared<addPeerOrder> (m_counter.getRpcId(),peer);
```
Comment (mikurys) [inline src-gui/commandsender.cpp]: bad alignment

File: src-gui/commandsender.cpp [hunk 5]
```diff
@@ -99,19 +99,19 @@ std::shared_ptr<order> CommandSender::prepareCommand(CommandSender::orderType ty
...
         break;
 
         case orderType::DELETEPEER:
         if(m_client_name.size() == 0)
             throw std::runtime_error ("no client name provided - can't send command");
-            ord = std::make_shared<deletePeerOrder> (new deletePeerOrder(m_counter.getRpcId(),peer));
+			ord = std::make_shared<deletePeerOrder> (m_counter.getRpcId(),peer);
```
Comment (mikurys) [inline src-gui/commandsender.cpp]: as above

File: src-gui/commandsender.cpp [hunk 6]
```diff
@@ -99,19 +99,19 @@ std::shared_ptr<order> CommandSender::prepareCommand(CommandSender::orderType ty
...
         break;
 
         case orderType::BANPEER:
             if(m_client_name.size() == 0)
             throw std::runtime_error ("no client name provided - can't send command");
-            ord = std::make_shared<banPeerOrder> (new banPeerOrder(m_counter.getRpcId(),peer));
+			ord = std::make_shared<banPeerOrder> (m_counter.getRpcId(),peer);
```
Comment (mikurys) [inline src-gui/commandsender.cpp]: as above

File: src-gui/mainwindow.cpp [hunk 7]
```diff
@@ -309,18 +309,21 @@ void MainWindow::loadSettings()
...
 {
     QSettings setings;
-    if(setings.allKeys().size() >= 2) {			//nie trzeba inicializowac
-        return;
-    }
+//    if(setings.allKeys().size() >= 2) {			//nie trzeba inicializowac
+//        return;
+//	}
+
+	if( setings.contains("rpcConnection/Ip"))
```
Comment (mikurys) [inline src-gui/mainwindow.cpp]: bad alignment

File: src-gui/order.cpp [hunk 8]
```diff
@@ -172,13 +172,13 @@ addPeerOrder::addPeerOrder(const std::string &json_str,commandExecutor * executo
         using nlohmann::json;
         json j = json::parse(json_str);
         m_executor = executor;
-        m_msg = j["msg"];
+			m_re = j["re"];
```
Comment (mikurys) [inline src-gui/order.cpp]: bad alignment

File: src-gui/order.cpp [hunk 9]
```diff
@@ -196,7 +196,7 @@ std::string addPeerOrder::get_str() const
 void addPeerOrder::execute(MainWindow &main_window)
 {
     if(m_state.find("error") != std::string::npos){ //if contains error
-        main_window.errorNotification(QString::fromStdString(m_msg));
+		main_window.errorNotification(QString::fromStdString(m_msg));
```
Comment (mikurys) [inline src-gui/order.cpp]: bad alignment

File: src-gui/peersmodel.h [hunk 10]
```diff
@@ -26,7 +26,7 @@ class peersModel : public QAbstractListModel
 
     // Basic functionality:
     int rowCount(const QModelIndex &parent = QModelIndex()) const override;
-    int columnCount(const QModelIndex &parent = QModelIndex()) const
+	int columnCount(const QModelIndex &parent = QModelIndex()) const override
```
Comment (mikurys) [inline src-gui/peersmodel.h]: bad alignment

**Comment by woskov** [comment]
will be replaced by #434

**Pull request closed without merging**